standard/table.lua

```diff
@@ -27,11 +29,13 @@ end
 
 ---@param tbl table
 ---@param value any
+---@param isPattern boolean?
 ---@return boolean
-function Table.includes(tbl, value)
+function Table.includes(tbl, value, isPattern)
 	for _, entry in pairs(tbl) do
-		if entry == value then
-			return true
+		if isPattern and Logic.isNumeric(string.find(entry, value))
```

Comment:
we do not need the `Logic.isNumeric` here, right?
like `string.find(entry, value)` returns nil and nill is treated as false in this context